ludwig/modules/reduction_modules.py

```diff
@@ -73,34 +91,9 @@ def dont_reduce(sequence, **kwargs):
     'avg': reduce_mean,
     'max': reduce_max,
     'concat': reduce_concat,
-    'attention': reduce_feed_forward_attention,
+    'attention': FeedForwardAttentionReducer(),
+    'null': dont_reduce,  # todo tf2 do we need this value,added to support text_feature()
```

Comment:
no we don't need it, let's modify to another valid value.

Reply:
OK...I'll modify `text_feature` to use one of the existing keys, e..g., `None`